tests/astronomer-scc-privileged_test.yaml

```diff
@@ -0,0 +1,20 @@
+---
+suite: Test astronomer-scc-privileged
```

Comment:
let's rewrite it with pytest.
@danielhoherd wdyt?

Reply:
I think this PR isn't needed anymore. We now have [`privileged: true` in master](https://github.com/astronomer/astronomer/blob/05f14f4570/charts/elasticsearch/templates/client/es-client-deployment.yaml#L78) and afaik we do not want `runAsUser: 0`, though maybe that is different with openshift. @pgvishnuram what do you think?

Reply:
I think we would want to solve this with `fsGroup`, not `runAsUser: 0`